lib/ReactViews/FeatureInfo/FeatureInfoPanel.jsx

```diff
@@ -200,8 +204,9 @@ function determineCatalogItem(nowViewing, feature) {
         }
 
         for (i = nowViewing.items.length - 1; i >= 0; i--) {
-            if (nowViewing.items[i].dataSource === dataSource) {
-                result = nowViewing.items[i];
+            item = nowViewing.items[i];
+            if (item.dataSource === dataSource || (defined(item._geoJsonItem) && item._geoJsonItem.dataSource === dataSource)) {
```

Comment:
Aha, that makes sense. I agree, might be good to do that. Other than that I think this looks fine :+1: